test/ibmq/test_ibmq_backends.py

```diff
@@ -105,11 +103,10 @@ def test_qobj_headers_in_result_sims(self, qe_token, qe_url):
             with self.subTest(backend=backend):
                 circuits = transpile(self.qc1, backend=backend)
 
-                # Update the Qobj header.
-                qobj_header = QobjHeader.from_dict(custom_qobj_header)
                 # TODO: assemble appends extra keys to the header in terra 0.8.
```

Comment:
Can you also delete the line with the TODO?

Reply:
Done.

Reply:
Great, thanks!